### ISSUE TITLE
Suggestion -- put author bio's in separate folder and use includes to add them to articles

### ISSUE THREAD

**Issue by meker12** [rackerlabs/docs-container-service#408]
Mike Metral wrote all the Best Practices articles.  We have the same bio in quite a few articles.  Would be better to include by reference.  Also, keeping the bios separate would make it easier for authors to change or update if required.

**Comment by rose-coste** [comment]
Yes, I hope at some point we'll centrally manage author info like at http://blog.rackspace.com/author/adrian-otto/ and https://developer.rackspace.com/blog/authors/mike-metral/, but it didn't seem like we'd get there for this release.
